### DISCUSSION TITLE
[Feature] <add PT-BR translation>

### DISCUSSION THREAD

**Discussion by maaxpane** [immich-app/immich#9268] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Good afternoon, I would like to thank you for adding the language choice option, but PT-BR is missing, I would like to help with the translation if necessary, I await contact, a suggestion if it is possible to choose the language on the web would also be great, thank you

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#9269
